docs/jwst/assign_wcs/main.rst

```diff
@@ -97,11 +98,20 @@ For each exposure, assign_wcs uses reference files and WCS header keywords
 to create the WCS object. What reference files are retrieved
 from CRDS is determined based on EXP_TYPE and other keywords in the science file header.
 
+The assign_wcs step can accept the single slope image that is the result of averaging
+over all integrations or a cube in the case of TSO observations.
 
-The assign_wcs step can accept any type of DataModel as input. In particular, for
-multiple-integration datasets the step will accept either of these data products:
-the slope results for each integration in the exposure, or the single slope image
-that is the result of averaging over all integrations.
+WFSS mode
+---------
+
+The WCS forward transforms for WFSS observations (``NIS_WFSS``, ``NRC_WFSS``, ``NRC_TSGRISM``)
+take as input the ``x, y`` coordinates on the dispersed image, the ``x0, y0`` coordinate of
+the center of the object in the direct imahe and ``spectral order``. They return the ``x0, y0`` coordinate of the center
+of the object in the direct image, ``wavelength`` and ``spectral order``.
+
+For NIRISS WFSS data the reference files contains a reference value for the filter wheel
```

Comment:
"files contains" are inconsistent. One of them needs to lose the "s" on the end: either "file contains" or "files contain", depending on whether there's more than one reference file.